Source/Plugin.BLE/Plugin.BLE.csproj

```diff
@@ -2,7 +2,7 @@
 	<PropertyGroup>
 		<TargetFrameworks>netstandard2.0;net7.0-android33.0;net8.0-android34.0</TargetFrameworks>
 		<TargetFrameworks Condition="!$([MSBuild]::IsOSPlatform('linux'))">$(TargetFrameworks);net7.0-ios;net7.0-maccatalyst;net8.0-ios;net8.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);MonoAndroid13.0;Xamarin.iOS10;Xamarin.Mac20;uap10.0.19041;net7.0-windows10.0.19041;net8.0-windows10.0.19041</TargetFrameworks>
+		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows'))">$(TargetFrameworks);MonoAndroid13.0;Xamarin.iOS10;Xamarin.Mac20;uap10.0.19041;net7.0-windows10.0.19041;net8.0-windows10.0.19041;net8.0-windows10.0.22000.0</TargetFrameworks>
```

Comment:
Why do you add an additional Windows API version here, instead of updating the existing one? We anyway support Windows versions down to 10.0.17763.0, right? In there a reason we need two net8.0-windows targets?

Also, why not do the same for .NET7? Currently, the net7.0 target would not be able to use the newer API features. If .NET7 supports API 10.0.222, I would recommend to update it as well ...

Reply:
I guess if we only have net8.0-windows10.0.22000.0 and not net8.0-windows10.0.19041, then if someone is on net8.0-windows10.0.19041, then they cannot use this package.  - and likewise for .Net7.0

The latest is in fact net8.0-windows10.0.22621.0, which I use

Reply:
I resolved by mistake I guess - and reopened. We don't have a conclusion.
I agree that it is currently compiled to many platforms - at least I simplified the codebase so we only have one Windows implementation now. As a NuGet provider do you then have any usage statistics of the platforms?

Reply:
> If we only have net8.0-windows10.0.22000.0 and not net8.0-windows10.0.19041, then if someone is on net8.0-windows10.0.19041, then they cannot use this package. - and likewise for .Net7.0

Ah, right. I missed that.


> You decide what is best for everybody - what I need is something greater or equal to net8.0-windows10.0.22000.0

I guess it's no big problem to have an additional target, but we have quite a number by now. At some point we may have to clean up some older ones.

Reply:
> As a NuGet provider do you then have any usage statistics of the platforms?

Unfortunately not. nuget.org only reports the download numbers of the packet, not on which platforms it's used.